polar_route/config_validation/vessel_schema.py

```diff
@@ -9,6 +9,10 @@
         "max_speed": {"type": "number", "minimum": 0},
         "unit": {"type": "string"},
         "max_ice_conc": {"type": "number", "minimum": 0, "maximum": 100},
-        "min_depth": {"type": "number", "minimum": 0}
+        "min_depth": {"type": "number", "minimum": 0},
+        "max_wave": {"type": "number", "minimum": 0},
```

Comment:
We need to do a more detailed testing for wave routing. For the time being remove "max_wave" until a later version of the codebase

Reply:
This is just an update to the vessel config schema to include the latest config parameters we have added since the schema was written. The `max_wave` parameter was actually added in #226 just over a year ago and works in the same way as e.g. the `max_ice_conc` param but using significant wave height. I don't think it makes any sense to not include it in the schema. It isn't currently included in the example config provided in the repo.